client-gateway/bot/.env.example

```
@@ -0,0 +1,13 @@
+# ========================================
+
+# Telegram Bot Token (required)
+# Obtain from @BotFather on Telegram
+BOT_TOKEN=your_telegram_bot_token_here
+
+# Service URLs (for future microservice integration)
+# TRIP_SERVICE_URL=http://trip-service:8080
+# DRIVER_SERVICE_URL=http://driver-service:8080
+
+# Logging level (DEBUG, INFO, WARNING, ERROR)
+LOG_LEVEL=INFO
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)